.gitignore

```diff
@@ -27,6 +27,7 @@
 .pub-cache/
 .pub/
 build/
```

Comment:
This will generate coverage reports or something else?

Reply:
yeah, so we need to make sure we don't commit coverage reports

Reply:
Sounds good. Note it to the team